<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>Hao_cen</author>
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>760</width>
    <height>280</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>760</width>
    <height>280</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>760</width>
    <height>280</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>崩坏3外置扫码器 v.1.4.1</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>0</y>
      <width>322</width>
      <height>271</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QFormLayout" name="formLayout_2">
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>登录B站账户</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QPushButton" name="loginBiliBtn">
         <property name="text">
          <string>点击登录</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>监听二维码</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QCheckBox" name="clipCheck">
         <property name="text">
          <string>当前状态:关闭</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>发送到手机端</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QCheckBox" name="broadcastCheck">
         <property name="text">
          <string>当前状态:关闭</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QTextBrowser" name="logText">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>431</width>
      <height>280</height>
     </rect>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'SimSun'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>250</y>
      <width>150</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Powered By Hao_cen</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>70</y>
      <width>269</width>
      <height>194</height>
     </rect>
    </property>
    <property name="text">
     <string>
简易使用说明：
第一次使用需要点击登录按钮登录B站账号
后续会将账号密码储存在配置文件内自动登录
请注意保护好文件安全

然后将监听二维码勾选上
这时候在剪贴板中的二维码图片将会自动识别
可以使用键盘上的PrintScreen按键快速截图</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>760</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>loginBiliBtn</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>login()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>621</x>
     <y>29</y>
    </hint>
    <hint type="destinationlabel">
     <x>454</x>
     <y>131</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>clipCheck</sender>
   <signal>clicked(bool)</signal>
   <receiver>MainWindow</receiver>
   <slot>qrCodeSwitch(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>545</x>
     <y>60</y>
    </hint>
    <hint type="destinationlabel">
     <x>447</x>
     <y>175</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>broadcastCheck</sender>
   <signal>clicked(bool)</signal>
   <receiver>MainWindow</receiver>
   <slot>broadcastSwitch(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>596</x>
     <y>86</y>
    </hint>
    <hint type="destinationlabel">
     <x>446</x>
     <y>246</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>login()</slot>
  <slot>qrCodeSwitch(bool)</slot>
  <slot>autoLoginSwitch(bool)</slot>
  <slot>broadcastSwitch(bool)</slot>
 </slots>
</ui>
